src/Microsoft.AspNet.Server.Kestrel/Http/Frame.cs

```diff
@@ -146,7 +185,8 @@ public void Reset()
 
             _prepareRequest?.Invoke(this);
 
-            _requestAbortCts?.Dispose();
+            _manuallySetRequestAbortToken = null;
+            _abortedCts = null;
```

Comment:
Don't we need to to dispose the last _abortedCts (if it exists) to clean up registrations?